include/dlaf/eigensolver/tridiag_solver/merge.h

```diff
@@ -41,26 +41,28 @@ namespace dlaf::eigensolver::internal {
 
 // Auxiliary matrix and vectors used for the D&C algorithm
 //
-// - mat1 and mat2: (matrices)
-//   Extra workspace for Q1', Q2' and U1' if n2 > n1 or U2' if n1 >= n2 which are packed as follows:
+// - e0: (matrix)
+//     In: Holds the eigenvectors of the two subproblems TODO in ascending order.
+//     Out: Holds the eigenvectors of the merged problems TODO in ascending order.
```

Comment:
`TODO`?

Reply:
Sorry, I added it as the documentation was not holding anymore, and forgot to update it...
https://github.com/eth-cscs/DLA-Future/pull/858/commits/79e3475a7b2a6f4fb77c1af09909a19d7fb1ec00